src/serialmonitor/serialMonitor.ts

```diff
@@ -77,6 +90,12 @@ export class SerialMonitor implements vscode.Disposable {
         this._baudRateStatusBar.command = "arduino.changeBaudRate";
         this._baudRateStatusBar.tooltip = "Baud Rate";
         this._baudRateStatusBar.text = defaultBaudRate.toString();
+
+        this._timestampFormatStatusBar = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right,
+                                                                           constants.statusBarPriority.TIMESTAMP_FORMAT);
+        this._timestampFormatStatusBar.command = "arduino.changeTimestampFormat";
+        this._timestampFormatStatusBar.tooltip = "Timestamp Format";
+        this._timestampFormatStatusBar.text = defaultTimestampFormat;

```

Comment:
You might consider changing this to a codicon rather than the defaultTimestampFormat. Currently, there's a chance that the button has no text if there is an empty timestamp. 
Something like $(watch) could work, or anything that represents time well.

Reply:
I searched through more [icons](https://code.visualstudio.com/api/references/icons-in-labels), but I think, that suggested `$(watch)` is the best option.

Changed in https://github.com/microsoft/vscode-arduino/pull/1430/commits/0628fb09e3a44ed530a4df8d45c9bb25eeab6221.